build.gradle.kts

```diff
@@ -1,8 +1,8 @@
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "3.1.7"
-  kotlin("plugin.spring") version "1.4.32"
-  kotlin("plugin.jpa") version "1.4.32"
-  kotlin("plugin.allopen") version "1.4.32"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "3.3.0"
```

Comment:
This essentially picks up the OWASP fix (a bump to the Spring Boot version (the plugin also uses Kotlin 1.5).